sc/src/main/java/uk/ac/ebi/atlas/home/LatestExperimentsService.java

```diff
@@ -39,6 +41,10 @@ public LatestExperimentsService(LatestExperimentsDao latestExperimentsDao,
         this.experimentTrader = experimentTrader;
     }
 
+    public String fetchLatestExperimentsJSON(){
```

Comment:
Any new public methods should be tested. To test services, please mock the DAO and trader and write some unit tests.